upgrade-notes/upgrade_notes.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+if ($argc < 3) {
+    echo "Error: version or upgrade-notes path is not set. Pass the version and the path as arguments.\n";
+    exit(1);
+}
+
+$version = $argv[1];
+$upgradeNotesPath = $argv[2];
```

Comment:
I would use `__DIR__` directly instead of specifying the dir in build.xml. 
I don't suppose we want to merge from another location

Reply:
yes, that sounds about right